README.md

```diff
@@ -1 +1,37 @@
-# awesome-books-project
\ No newline at end of file
+# Capstone-project-gym
```

Comment:
- [ ] You have done an excellent job on this project, congrats! Please update the README file info as this is a key part of your project documentation and you will be ready!  🚀